workers/email_reports/looping_ratio_decrease_alert/data.go

```diff
@@ -93,108 +59,12 @@ func compareResults(amDomainData map[string][]email_reports.AggregatedReport, ye
-
-	if err != nil {
-		return nil, fmt.Errorf("error marshalling request data, %w", err)
-	}
-
-	reportData, err := compassClient.Request("/report-dashboard/report-new-bidder", "POST", data, true)
-
-	if err != nil {
-		return nil, fmt.Errorf("error getting report data,%w", err)
-	}
-
-	var report email_reports.Report
-	err = json.Unmarshal(reportData, &report)
-	if err != nil {
-		return nil, fmt.Errorf("error unmarshalling report data: %w", err)
-	}
-
-	aggregatedReports := prepareLRReport(report)
-
-	return aggregatedReports, nil
-}
-
-func prepareLRReport(report email_reports.Report) []email_reports.AggregatedReport {
-	aggregatedReports := make([]email_reports.AggregatedReport, len(report.Data.Result))
-	formatter := &helpers.FormatValues{}
-	for i, r := range report.Data.Result {
-		if r.PubImps >= PubImpsThreshold {
-			aggregatedReports[i] = email_reports.AggregatedReport{
-				Date:                 r.Date,
-				DataStamp:            r.DataStamp,
-				Publisher:            r.Publisher,
-				Domain:               r.Domain,
-				PaymentType:          r.PaymentType,
-				AM:                   r.AM,
-				PubImps:              formatter.PubImps(int(r.PubImps)),
-				PublisherBidRequests: formatter.BidRequests(float64(r.PublisherBidRequests)),
-				LoopingRatio:         helpers.RoundFloat(r.LoopingRatio),
-				Ratio:                helpers.RoundFloat(r.Ratio),
-				CPM:                  helpers.RoundFloat(r.CPM),
-				Cost:                 helpers.RoundFloat(r.Cost),
-				RPM:                  helpers.RoundFloat(r.RPM),
-				DpRPM:                helpers.RoundFloat(r.DpRPM),
-				Revenue:              helpers.RoundFloat(r.Revenue),
-				GP:                   helpers.RoundFloat(r.GP),
-				GPP:                  helpers.RoundFloat(r.GPP),
-			}
-		}
-	}
-
-	return aggregatedReports
-}
-
-func getRequestData() email_reports.RequestData {
-	currentTime := time.Now().In(email_reports.Location)
-	startOfLast7Days := time.Date(currentTime.Year(), currentTime.Month(), currentTime.Day(), 0, 0, 0, 0, currentTime.Location()).AddDate(0, 0, -7)
-
-	endOfLast7Days := time.Date(currentTime.Year(), currentTime.Month(), currentTime.Day(), 23, 59, 59, 0, currentTime.Location()).AddDate(0, 0, 0)
-
-	startOfLast7DaysStr := startOfLast7Days.Format(time.DateTime)
-	endOfLast7DaysStr := endOfLast7Days.Format(time.DateTime)
-
-	requestData := email_reports.RequestData{
-		Data: email_reports.RequestDetails{
-			Date: email_reports.Date{
-				Range: []string{
-					startOfLast7DaysStr,
-					endOfLast7DaysStr,
-				},
-				Interval: "day",
-			},
-			Dimensions: []string{
-				"Publisher",
-				"Domain",
-				"AM",
-				"PaymentType",
-			},
-			Metrics: []string{
-				"PublisherBidRequests",
-				"nbLR",
-				"nbRatio",
-				"nbRpm",
-				"nbDpRpm",
-				"Revenue",
-				"nbGp",
-				"nbGpp",
-				"PubImps",
-				"nbCpm",
-				"Cost",
-			},
-		},
+	avgDataMap := make(map[string][]AlertsEmails)
+	for _, repo := range emailReports {
```

Comment:
What about different variable name? Repo sounds not related to `report`
```suggestion
	for _, report := range emailReports {
```

Reply:
done